examples/hydrogen-vqe.yaml

```diff
@@ -0,0 +1,198 @@
+# Workflow API version
+apiVersion: io.orquestra.workflow/1.0.0
+
+# Prefix for workflow ID
+name: vqe-h2-example
+
+imports:
+- name: z-quantum-core
+  type: git
+  parameters:
+    repository: "git@github.com:zapatacomputing/z-quantum-core.git"
+    branch: "workflow-v1"
```

Comment:
I think we want to have all branches of examples in master - see the guidelines. However, it's worth testing the workflow on the these branches to see if it works

Reply:
Addressed: 68f6fc1. Workflow is not  yet working  on `master` branches due to the release schedule